zqd/space/manager.go

```diff
@@ -44,37 +45,49 @@ func NewManager(root string, logger *zap.Logger) (*Manager, error) {
 			continue
 		}
 
-		space, err := loadSpace(path, config)
+		spaces, err := loadSpaces(path, config)
```

Comment:
Because subspaces do not have configs stored in their path you will get a logger error "Error loading config" for every subspace on manager open. We could change `loadConfig` to not log an error if config file DoesNotExist, that said I think my preference would be for a subspace to have its config in its path and for a subspace config to include a reference to its parentId. One of the nice things about this solution is if I needed to debug a space and all I had was the file system I could at least cat a space's config file and have some idea what it does.

Did not quite know where to put this comment: The fact that a sub-spaces configs are stored in its parent is a little strange to me. With this setup there will be

Reply:
You're correct that a does subspace in this PR does not have a config.json file for itself. Note that they also don't have a separate directory under zqd's root, so the error you mention won't occur.

The reason I went this route is to avoid needing to update file-based configuration in two places when you create a subspace, and to help prevent a parent space from being deleted when there are subspaces that refer to it.

Reply:
> Note that they also don't have a separate directory under zqd's root, so the error you mention won't occur.

I was missing this part. This sounds good then.